src/app/components/common/__snapshots__/Footer.spec.js.snap

```diff
@@ -0,0 +1,3 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`Footer component should be render correctly 1`] = `ShallowWrapper {}`;
```

Comment:
Your snapshot tests don't work. You need to add enzyme to snapshot serialized in jest config to make it render correctly